example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -385,32 +409,36 @@ describe('The S3 Ingest Granules workflow', () => {
       });
 
       it('allows reingest and executes with success status', async () => {
-        granule = await apiTestUtils.getGranule({
+        const firstGranuleResponse = await apiTestUtils.getGranule({
```

Comment:
It seems like this granule response and the one in `beforeAll` are returning the granule from the same execution, so they can probably be combined to reduce duplication?